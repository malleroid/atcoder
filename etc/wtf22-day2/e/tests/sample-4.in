20
167660508 377125547 866003430 419036363 234113368 201296307 408194538 249252693 207212853 190504619 306027011 652875643 335898069 545795899 204268068 274532279 342039277 975300308 901270584 360957186
