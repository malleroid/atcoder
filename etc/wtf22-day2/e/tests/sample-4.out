536870912
1610612736
2684354560
3758096384
4831838208
4831838208
4831838208
4831838208
4831838208
5100273664
5100273664
5100273664
5100273664
5100273664
5100273664
5100273664
5100273664
5100273664
5100273664
5100273664
